### ISSUE TITLE
Possibe to export multiple playlists?

### ISSUE THREAD

**Issue by andrejohansson** [xteve-project/xTeVe#146]
**Is your feature request related to a problem? Please describe.**
So far, I understand that xteve can take multiple playlists as input, and always returns one playlist as output (eg http://192.168.1.186:34400/m3u/xteve.m3u).

Is it somehow possible to create mulitple output playlists? 

**Describe the solution you'd like**
 I would like to have different playlist outputs for different persons in the home. Or is this achievable by some other way or organizing?

![ISetup](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggVERcbiAgQVtQcm92aWRlcl0gLS0-fFByb3ZpZGVyIHBsYXlsaXN0IHdpdGggY3JlZGVudGlhbHwgQ3tYdGV2ZX1cbiAgQyAtLT58TXkgc3Vic2V0IG9mIGNoYW5uZWxzfCBEWzxodHRwOi8vMTkyLjE2OC4xLjE4NjozNDQwMC9tM3UvbWUubTN1Pl1cbiAgQyAtLT58V2lmZSBzdWJzZXQgb2YgY2hhbm5lbHN8IEVbPGh0dHA6Ly8xOTIuMTY4LjEuMTg2OjM0NDAwL20zdS93aWZlLm0zdT5dXG4gIEMgLS0-fEtpZHMgc3Vic2V0IG9mIGNoYW5uZWxzfCBGWzxodHRwOi8vMTkyLjE2OC4xLjE4NjozNDQwMC9tM3Uva2lkcy5tM3U-XVxuXHRcdCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)

**Comment by mar-mei** [comment]
No, all users get the same playlist. However, you can start several xTeVe instances and then configure them separately.

**Comment by alturismo** [comment]
or may u could abuse the group title feature in xteve

![image](https://user-images.githubusercontent.com/8406490/92301649-9cd28880-ef65-11ea-91bf-567bb1b7f0d3.png)

will filter then like this
http://192.168.1.67:34400/m3u/xteve.m3u?group-title=wife
http://192.168.1.67:34400/m3u/xteve.m3u?group-title=kids
